src/domains/ecommerce-common/03_storage.tf

```diff
@@ -302,70 +302,128 @@ resource "azurerm_monitor_diagnostic_setting" "ecommerce_transient_queue_diagnos
 locals {
   queue_transient_alert_props = var.env_short == "p" ? [
     {
-      "queue_key"   = "transactions-expiration-queue"
+      "queue_key"   = "transaction-notifications-queue"
       "severity"    = 1
       "time_window" = 30
       "frequency"   = 15
       "threshold"   = 10
     },
     {
-      "queue_key"   = "transaction-notifications-queue"
+      "queue_key"   = "transactions-close-payment-queue"
       "severity"    = 1
       "time_window" = 30
       "frequency"   = 15
       "threshold"   = 10
     },
     {
-      "queue_key"   = "notifications-service-retry-queue"
+      "queue_key"   = "transactions-refund-queue"
       "severity"    = 1
       "time_window" = 30
       "frequency"   = 15
       "threshold"   = 10
-    },
+    }
+  ] : []
+}
+
+# Queue size: Ecommerce - ecommerce queues enqueues rate alert
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_transient_enqueue_rate_alert" {
+  for_each            = { for q in local.queue_transient_alert_props : q.queue_key => q }
+  name                = "${local.project}-${each.value.queue_key}-rate-alert"
+  resource_group_name = azurerm_resource_group.storage_ecommerce_rg.name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id, azurerm_monitor_action_group.ecommerce_opsgenie[0].id]
+    email_subject          = "Email Header"
```

Comment:
Change this email_subject field to reflect the alert reason.
Please change it also for the below alert too